ldpd/ldpd.c

```diff
@@ -228,8 +230,9 @@ main(int argc, char *argv[])
 		saved_argv0 = (char *)"ldpd";
 
 	frr_preinit(&ldpd_di, argc, argv);
-	frr_opt_add("LEn:", longopts,
+	frr_opt_add("LESn:", longopts,
 		"      --ctl_socket   Override ctl socket path\n"
+		"  -S, --skip_runas   Skip capabilities checks, and changing user and group IDs.\n"
```

Comment:
Skip which capabilities checks? With this new option ldpd still needs root privileges in order to run.